README.md

```diff
@@ -39,16 +45,26 @@ We also have a set of [examples](https://github.com/beehive-lab/TornadoVM/tree/m
 
 [Profiler Usage](assembly/src/docs/9_PROFILER.md)
 
-
 ## 3. Programming Model
 
-TornadoVM exposes to the programmer task-level, data-level and pipeline-level parallelism via a light Application Programming Interface (API). TornadoVM uses single-source property, in which the code to be accelerated and the host code live in the same Java program.
+TornadoVM exposes to the programmer task-level, data-level and pipeline-level parallelism via a light Application
+Programming Interface (API). TornadoVM uses single-source property, in which the code to be accelerated and the host
+code live in the same Java program.
+
+Compute-kernels in TornadoVM can be programmed using two different APIs:
+
+#### a) Loop-parallelism
+
+Compute kernels are written in a sequential form (task for a single processors). To express parallelism, TornadoVM
```

Comment:
i dont understand the "task for a single processors"

Reply:
I wanted to express a task for single thread. I will change this.

Reply:
I think the word `task` causes confusion, as it is not clear what is the task.

Should it be `(a single task for a CPU thread)`?